Readme.md

```diff
@@ -409,10 +409,10 @@ jobs:
 
 ### 10. Más Locators
 
-**Descripción**: esta sesión automatizaremos otra página diferente, y su misión es seleccionar los mejores locators posibles de tal forma que el page object sea lo más reutilizable posible
+**Descripción**: En esta sesión automatizaremos otra página diferente, y su misión es seleccionar los mejores locators posibles de tal forma que el page object sea lo más reutilizable posible
 
-1. Crear el archivo **personal-information.page.ts** en la carpeta src/page
-1. Crear el archivo **locators.spec.ts** en la carpeta de test, dentro de este archivo se navegará a <http://toolsqa.com/automation-practice-form/> y ejecutará el siguiente método que debe llenar el formulario con la información que se indica y dar clic en el botón Button (Evitar el uso de css locators)
+1. Crear el archivo **personal-information.page.ts** en la carpeta integration/page
```

Comment:
I move the page objects to the folder cypress/page, this to avoid loading the page object files in the cypress test runner, change integration/page to cypress/page